app/views/docs/migrations-cloud-to-local.phtml

```diff
@@ -0,0 +1,34 @@
+<p>
```

Comment:
This file is called `cloud-to-local`, why not `cloud-to-self-hosted`? If you decide to change, make sure to also update all links here and in appwrite/homepage